pkg/workflow_template.go

```diff
@@ -498,6 +552,27 @@ func (c *Client) CreateWorkflowTemplateVersion(namespace string, workflowTemplat
 	return workflowTemplate, nil
 }
 
+// UpdateWorkflowTemplateVersionDB will update a given WorkflowTemplateVersion in the database.
+// The intent is to change specific database values for a WorkflowTemplateVersion.
+func (c *Client) UpdateWorkflowTemplateVersionDB(namespace string, wtv *WorkflowTemplateVersion) error {
+	if wtv.ID == 0 {
+		return fmt.Errorf("id required for UpdateWorkflowTemplateVersionDB")
+	}
+
+	tx, err := c.DB.Begin()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback()
+
+	err = updateWorkflowTemplateVersionDB(tx, wtv)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
@aleksandrmelnikov 

Looks like namespace is not being used, can we delete it from arguments?

Also, since `updateWorkflowTemplateVersionDB` uses a `sq.BaseRunner` you can pass in `c.DB` as the argument instead of tx.

Finally, this may be more of my preference, but if you name the return as 
```go
(err error)
```

You can omit the err check and have
```go
err = updateWorkflowTemplateVersionDB(c.DB, wtv)

return
```

I find it a little cleaner.

Reply:
Agreed